drivers/hmis/app/models/hmis/filter/enrollment_filter.rb

```diff
@@ -55,56 +55,63 @@ def with_household_tasks(scope)
     with_filter(scope, :household_tasks) do
       return scope.all unless input.household_tasks&.include?('ANNUAL_DUE')
 
+      # SQL-ized version of the logic here: drivers/hmis/app/models/hmis/reminders/reminder_generator.rb#annual_assessment_reminder
+
       # The SQL statement "interval"((extract(day from "EntryDate")::integer - 1) || ' days') is present to handle leap
       # year dates. That is, if the entry date is 2/29/2020, then just using make_date directly will produce an
       # anniversary date of 2/29/2023, which is invalid. To prevent this, make_date is used to create a date of
       # 2/1/2023, then we add 28 days (29th - 1) to that date to shift it to the proper anniversary date. For any date
       # except for 2/29, this shift should result in the same month and day as the entry date, and for 2/29 it should
       # shift it to 3/1, which is a valid date and avoids the leap year issue.
 
-      # Start of the 60-day window during which this year's Annual should be performed. (Anniversary - 30 days)
-      start_date = Arel.sql <<~SQL
+      anniversary_date = "
```

Comment:
@gigxz any reason we switched away from a heredoc?  They tend to be more resilient for SQL since they allow both kinds of quotes without escaping.

Reply:
Hmm, not sure about interpolation, I haven't noticed an issue before. Looks like at some point in history it was possible. https://medium.com/rubycademy/avoid-interpolation-in-heredoc-59a5d907133d

Reply:
This has some more details. https://blog.saeloun.com/2020/04/08/heredoc-in-ruby-and-rails/

Reply:
oh I got it working now, thanks!